src/services/navigationBar.ts

```diff
@@ -154,6 +154,20 @@ namespace ts.NavigationBar {
             for (let node of nodes) {
                 switch (node.kind) {
                     case SyntaxKind.ClassDeclaration:
+                        topLevelNodes.push(node);
+                        forEach((<ClassDeclaration>node).members, (node) => {

```

Comment:
Use `for-of` over forEach unless you're unsure whether the first argument is `undefined`